server.js

```diff
@@ -7,10 +7,57 @@ app.use(express.json())
 app.set('port', process.env.PORT || 3000);
 app.locals.title = 'Palette Picker';
 
+/* GET REQUESTS */
 app.get('/', (request, response) => {
   response.status(200).json('Initial Setup Complete');
 });
 
+app.get('/api/v1/projects', (request, response) => {
+  response.status(200).json('Initial Setup Complete for Projects');
+});
+
+app.get('/api/v1/projects/:id', (request, response) => {
+  response.status(200).json('Initial Setup Complete for Individual Project');
+});
+
+app.get('/api/v1/palattes', (request, response) => {
+  response.status(200).json('Initial Setup Complete for Palettes');
+});
+
+app.get('/api/v1/palettes/:id', (request, response) => {
+  response.status(200).json('Initial Setup Complete for Individual Palette');
+});
+
+/* POST REQUESTS */
+
+app.post('/api/v1/projects', (request, response) => {
+  response.status(201).json('Initial Setup Complete for Adding Project');
+})
+
+app.post('/api/v1/projects', (request, response) => {
```

Comment:
this one should say palettes and not projects.

Reply:
Corrected path to palettes